Cabal-hooks/CHANGELOG.md

```diff
@@ -1,12 +1,9 @@
-# Changelog for `Cabal-hooks`
-
-## 3.16 – November 2024
-
-  * Release accompanying `Cabal-3.16`.
-    No changes to `Cabal-hooks` itself; all changes come changes in re-exports
-    from `Cabal`.
-
-## 3.14 – November 2024
-
-  * Initial release of the `Hooks` API.
-
+# Changelog for `Cabal-hooks`
```

Comment:
If we have no changes and we reverted the revbump on its Cabal file, do we even need this entry?

Reply:
On second thought, this is the forward-port, so that ship has sailed, I guess?

Reply:
Took me a while to decode what you first said but I agree that it's an issue. Let me speak up on Matrix...

In the meantime, an approval would be welcome!